### ISSUE TITLE
ClipOval of CameraPreview inside Positioned - possible layout bug

### ISSUE THREAD

**Issue by mwk24** [flutter/flutter#41796]
## Steps to Reproduce

Here is the code in question (inside a fairly vanilla Stack). Note the left, top props of Positioned.

```
                    // WILL APPEAR
                    Positioned(
                      left: 99,
                      top: 177,
                      child: Container(
                        height: 100,
                        width: 100,
                        child: ClipOval(
                          child: OverflowBox(
                            maxWidth: 100,
                            maxHeight: 100 / _cameraController.value.aspectRatio, //(=178)
                            child: CameraPreview(_cameraController),
                          ),
                        ),
                      )
                    ),
                    
                    // WILL NOT APPEAR - seems when either left or top >= width or height it won't appear
                    // If I switch the CameraPreview for a Container, it works fine
                    Positioned(
                      left: 100,
                      top: 178,
                      child: Container(
                        height: 100,
                        width: 100,
                        child: ClipOval(
                          child: OverflowBox(
                            maxWidth: 100,
                            maxHeight: 100 / _cameraController.value.aspectRatio, //(=178)
                            child: CameraPreview(_cameraController),
                          ),
                        ),
                      )
                    ),

```

It seems like if left or top of the Positioned is greater than the size of the CameraPreview, it won't appear. I replaced CameraPreview with another widget and it works. Perhaps something to do with the behavior of Texture, and perhaps I'm misunderstanding how it is inheriting constraints. 

Either it's a bug or it's very confusing.

flutter analyze: [no errors]

```
[✓] Flutter (Channel master, v1.10.7-pre.78, on Mac OS X 10.14.5 18F132, locale en-US)
    • Flutter version 1.10.7-pre.78 at /Users/mark/dev/flutter
    • Framework revision 839fdbd2ed (3 days ago), 2019-09-29 21:32:06 -0700
    • Engine revision 5b952f286f
    • Dart version 2.6.0 (build 2.6.0-dev.0.0 6ff8d2199d)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/mark/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.0)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.0, Build version 11A420a
    • CocoaPods version 1.7.2

[✓] Android Studio (version 3.4)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 39.0.1
    • Dart plugin version 183.6270
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)

[✓] VS Code (version 1.38.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.4.1

[✓] Connected device (1 available)
    • Mark Kinsey’s iPhone • 6288e874eac8ce4397e492ef22e5aa9d29aa2bae • ios • iOS 13.1

• No issues found!
````

**Comment by BondarenkoStas** [comment]
@mwk24 Can you please try it on another emulator/device, to learn if it's ios13 specific or not?

**Comment by mwk24** [comment]
Actually, can't use emulator as it doesn't have camera, and I don't have an android drive handy. I might have one lying around, will check.

**Comment by mwk24** [comment]
* Proceeds to debug Gradle for 7 hours

**Comment by mwk24** [comment]
Tested on physical android device. Same issue (though the Positioned params to make it break are different because of the different aspect ratio I believe)

**Comment by goderbauer** [comment]
WHere's CameraPreview coming from? That's not part of the framework.

Can you post a runnable reproduction of your problem? Something we can copy and paste and run to see the actual problem?

**Comment by mwk24** [comment]
CameraPreview is from the camera plugin made by flutter team. Will try to put together runnable gist

**Comment by mwk24** [comment]
<script src="https://gist.github.com/mwk24/e5670368aafa474cefa2e05292c4cfc2.js"></script>

**Comment by mwk24** [comment]
[<script src="https://gist.github.com/mwk24/e5670368aafa474cefa2e05292c4cfc2.js"></script>](url)

**Comment by mwk24** [comment]
https://gist.github.com/mwk24/e5670368aafa474cefa2e05292c4cfc2

**Comment by mwk24** [comment]
any updates? thanks!

**Comment by mwk24** [comment]
Would an appreciate an update or some ideas or perhaps a workaround. My project is close to being blocked on this. Thanks!

**Comment by mwk24** [comment]
This is now working after flutter update. Thanks for listening.
